CHANGELOG

```diff
@@ -1,3 +1,21 @@
+v1.5.5 (Mar 2023)
+fix: fix rare corruption bug affecting the high compression mode (#3517) by @terrelln
```

Comment:
Intentionally finding such corruption bug is greate, I would suggest to add discoverer's name. It seems he is a testing team member.

Reply:
Discoverer's name is planned to be part of the release note.

But indeed, it could be added to the changelog too.

Reply:
It's coming

Reply:
Not really a testing team member. Just the owner of ZSTD sync at Google and adviser on APIs, etc :)